lib/mindwendel/repo.ex

```diff
@@ -2,4 +2,8 @@ defmodule Mindwendel.Repo do
   use Ecto.Repo,
     otp_app: :mindwendel,
     adapter: Ecto.Adapters.Postgres
+
+  def count(query) do
+    aggregate(query, :count)
```

Comment:
just a nice little helper as I saw way too many `Repo.all(..) |> Enum.count` and this is better :) Also `length()` is shorter should one need to count